openslides/core/static/js/core/base.js

```diff
@@ -819,11 +819,13 @@ angular.module('OpenSlidesApp.core', [
                                 id: value.id,
                             };
                         } else if (
-                            value.name != 'agenda/item-list' &&
-                            value.name != 'core/clock' &&
-                            value.name != 'core/countdown' &&
-                            value.name != 'core/projector-message' &&
-                            value.name != 'agenda/current-list-of-speakers' ) {
+                            value.name === 'agenda/item' ||
```

Comment:
Ok, but please insert a comment here and a GitHub issue that we do not want to hardcode the apps in core here but implement a generic solution later (not for 2.1 of course).